apps/src/gamelab/animationListModule.js

```diff
@@ -177,7 +186,7 @@ export function addBlankAnimation() {
     dispatch({
       type: ADD_ANIMATION,
       key,
-      data: {
+      props: {
```

Comment:
This fixes the current bug in staging that breaks starting a new blank animation - it's a merge bug (data got renamed to props in one branch, but this was added in a different branch) that didn't get caught.  I need to get back to adding tests around all this stuff.